Packs/Palo_Alto_Networks_Security_Advisories/Integrations/PaloAltoNetworksSecurityAdvisories/PaloAltoNetworksSecurityAdvisories.yml

```diff
@@ -0,0 +1,175 @@
+category: Authentication
+commonfields:
+  id: Palo_Alto_Networks_Security_Advisories
+  version: -1
+configuration:
+- defaultvalue: https://security.paloaltonetworks.com/api/v1
+  display: Default URL for PAN-OS advisories website
+  name: url
+  required: false
+  type: 0
+- additionalinfo: The Palo Alto Networks product name to fetch indicators for, such
+    as "PAN-OS"
+  defaultvalue: PAN-OS
```

Comment:
@adambaumeister 3 thoughts:
1. why have by default PAN-OS and not all?
2. can we provide. a closed list of products(or a link to a doc)
3. can we select 2/3 out of X products? say "PAN-OS, XDR, Prisma Cloud"

Reply:
1. Product must be specified, it can't be empty or "all" as far as I am aware.
2. Yes but onyl via the products API; https://security.paloaltonetworks.com/api/v1/products
3. Nope - it's tied specifically to a product in the actual API endpoint. i.e has to  be /products/PAN-OS.

Reply:
thanks Adam.
can you commit the addition of the command (2) and elaborate in that in the additionalinfo?
If you can not commit to this PR, we can do an enhancement later, this is not time pressing.

Reply:
Yeah I actually can't add any commits into this PR - happy to do a subsequent PR at a later date for sure to add that command. I'll also reach out to the maintainer of this API to see what's on the roadmap in terms of additional API functionality here.